### PR TITLE
fixed typo in .travis.yml

### PR DESCRIPTION
**Pull request by flying-sheep**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ install:
 
 script:
   - flake8 aiohttp
-  - if python -c "import sys; sys.exit(sys.version_info >= (3,5)"; then
+  - if python -c "import sys; sys.exit(sys.version_info >= (3,5))"; then
```
Comment (popravich) [inline .travis.yml:27]: Heh, I believe it worked "as expected" with that typo)
But real check must look like this:

``` yaml
- if python -c "import sys; sys.exit(sys.version_info < (3,5))"; then
      flake8 examples tests;
  fi
```

`sys.exit()` must be called with `0 / False` for non-errored runs.

**Comment by popravich** [comment]
Please fix it!

**Comment by asvetlov** [comment]
@popravich please do it yourself

**Comment by popravich** [comment]
You merge too fast)
done in master

**Comment by flying-sheep** [comment]
eh well, i fixed the typo as advertized, i didn’t think about the logic :smile:

**Pull request merged**